docs/modules/nifi/pages/usage_guide/exposing-processors/http.adoc

```diff
@@ -0,0 +1,99 @@
+
+In this case `POST` requests are being send to NiFi, which acts as a webhook to receive data here.
+You should also be able to serve `GET` requests by using the `HandleHttpRequest` processor, however this is currently not demonstrated in this guide.
+
+== 1. Create `ListenHTTP` processor
+
+Let's start by creating a `ListenHTTP` processor:
+
+1. Set `Base Path` to an empty string
+2. Set `Listening Port` to `8042` or any other unused port number you prefer
+3. Set `Record Reader` to an `JsonTreeReader` and `Record Writer` to an `JsonRecordSetWriter` instance.
+   This guide assumes you are receiving JSON documents, but you can also receive other files types, please update the record reader and writer accordingly.
+
+You should end up with something similar to
+
+image:listen-http-1.png[A ListenHTTP processor]
+
+== 2. Expose `ListenHTTP` processor
+
+Afterwards you need to expose the processor the the outside world of NiFi.
+
+For that to work, first create a Service object as follows.
+This guide assumes your NifiCluster is called `simple-nifi`, please update the YAML accordingly.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: simple-nifi-listen-http # Update according to NifiCluster name
+spec:
+  type: ClusterIP
+  selector:
+    app.kubernetes.io/component: node
+    app.kubernetes.io/instance: simple-nifi # Update according to NifiCluster name
+    app.kubernetes.io/name: nifi
+  ports:
+  - name: http
+    port: 8042
+    protocol: TCP
+    targetPort: 8042
+----
+
+In case you don't have an ingress controller, you can set the Service type to `LoadBalancer` or `NodePort` instead and should be ready to go.
+If you are using an ingress controller, an Ingress could look something like
+
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: simple-nifi-listen-http # Update according to NifiCluster name
+spec:
+  rules:
+  - host: simple-nifi-listen-http.my.corp # Update to your host
+    http:
+      paths:
+      - pathType: Prefix
+        path: /
+        backend:
+          service:
+            name: simple-nifi-listen-http # Update to your Service name
+            port:
+              number: 8042
+----
+
+=== 3. Route 
+
+First, create a `RouteOnAttribute` processor and connect it to the `success` output of the `ListenHTTP` processor as shown below.
+Start the `ListenHTTP` processor.
+
+image:listen-http-2.png[´RouteOnAttribute connected to ListenHTTP processor]
+
+The `ListenHTTP` processor should now generate a FlowFile for every incoming HTTP request.
+You can test this e.g. by calling `curl --verbose --data '{"hello":"NiFi"}' https://simple-nifi-listen-http.my.corp`, you should get a `HTTP/2 200` response.
+
+TIP: If you get a `503 Service Temporarily Unavailable`, this probably means your ingress controller was not able to
+reach your `ListenHTTP` processor, it is probably not running or not configured correctly.
```

Comment:
```suggestion
TIP: If you get a `503 Service Temporarily Unavailable`, this probably means your Ingress controller was not able to
reach your `ListenHTTP` processor. Check that the Ingress controller is running and configured correctly.
```

Reply:
Maybe this should be a multi-line admonition, but I can never remember how to do is in asciidoc.

Reply:
I was actually referring to the ListenHTTP processor being badly configured here, not the Ingress controller.
The ingress controller is very likely working, as they can access the WebUI and it returns a 503 ;)

Reply:
Ok, maybe that should be clearer.

```suggestion
TIP: If you get a `503 Service Temporarily Unavailable`, this probably means your Ingress controller was not able to
reach your `ListenHTTP` processor. Check that the processor is running running and configured correctly.
```

Reply:
Thanks, I went with
TIP: If you get a `503 Service Temporarily Unavailable`, this probably means your Ingress controller was not able to
reach your `ListenHTTP` processor. Check that the processor is running and configured correctly.